docs/troubleshooting/other-troubleshooting-tips/rancher-ha.md

```diff
@@ -86,3 +86,20 @@ Example output:
 NAME                 HOLDER                    AGE
 cattle-controllers   rancher-dbc7ff869-gvg6k   6h10m
 ```
+
+#### Configuration
+
+_Available as of Rancher 2.8.3_
+
+In the unfortunate event of Kubernetes API latency degradation, the Rancher replica holding the leader lock may not be able to renew the lease within its validity duration.
+In order to mitigate this, it is possible to modify the default parameters for leader election using environment variables:
```

Comment:
```suggestion
If the Kubernetes API experiences latency, the Rancher replica holding the leader lock may not be able to renew the lease before the lease becomes invalid. To mitigate this, in Rancher v2.8.3 and later you can use environment variables to modify the default parameters for leader election:
```

Reply:
I think it's preferable for the version availability information to be highly visible instead of appearing in the middle of a paragraph where it can missed.

Reply:
+1, BTW I took the example from:
https://ranchermanager.docs.rancher.com/reference-guides/user-settings/user-preferences#confirmation-setting

I can apply the suggested change while keeping the banner.